src/app/pages/management/edit-project/edit-project.component.ts

```diff
@@ -2,18 +2,19 @@ import { Component, OnInit, ViewChild } from '@angular/core';
 import { Router, ActivatedRoute } from '@angular/router';
 import { DomSanitizer } from '@angular/platform-browser';
 
-import { ConfirmationService, Message, SelectItem } from 'primeng/primeng';
+import { ConfirmationService, Message, MessageService, SelectItem } from 'primeng/primeng';
 
 import {
-    APIError, User, Project, Site, Dataset, ModelChoice, Program
+    APIError, User, Project, Site, Dataset, ModelChoice, Program, ProjectMedia, Media
```

Comment:
Is it `Media` or `RecordMedia`?

Reply:
Okay just read the interface. `Media` is the super.

Reply:
The media manager emits media, because it's non-specific, so the callbacks need it in their signature.